pkg/services/sqlstore/migrations/accesscontrol/dashboard_permissions.go

```diff
@@ -394,6 +394,110 @@ func (m *managedFolderAlertActionsMigrator) Exec(sess *xorm.Session, mg *migrato
 	return nil
 }
 
+const managedFolderAlertActionsRepeatMigratorID = "managed folder permissions alert actions repeated migration"
+
+func AddManagedFolderAlertActionsRepeatMigration(mg *migrator.Migrator) {
+	mg.AddMigration(managedFolderAlertActionsRepeatMigratorID, &managedFolderAlertActionsRepeatMigrator{})
+}
+
+type managedFolderAlertActionsRepeatMigrator struct {
+	migrator.MigrationBase
+}
+
+func (m *managedFolderAlertActionsRepeatMigrator) SQL(dialect migrator.Dialect) string {
+	return CodeMigrationSQL
+}
+
+func (m *managedFolderAlertActionsRepeatMigrator) Exec(sess *xorm.Session, mg *migrator.Migrator) error {
```

Comment:
note: the migration logic is very similar to `managedFolderAlertActionsMigrator` defined above. The only difference is that here we check that a permission has not already been added before appending it to `toAdd` list.